### PR TITLE
Deref event for when the are manually passed to the lambda function.

### PR DESCRIPTION
**Pull request by Luke1298**

### CODE REVIEW

File: aws_lambda/app.py [hunk 1]
```diff
@@ -17,7 +17,9 @@
 
 
 def api_func(event, context):
-    if type(event) is dict and "headers" in event and "body" in event:
+    if type(event) is dict and "event" in event and (not "headers" in event and "body" in event):
+        event = event["event"] #Manually invoking lambda stuffs the event into the 'event' key
+    if type(event) is dict "headers" in event and "body" in event:
```
Comment (bradgrimm) [inline aws_lambda/app.py]: Is this missing an `and` after `dict`?

Comment (Luke1298) [inline aws_lambda/app.py]:
```suggestion
    if type(event) is dict and "headers" in event and "body" in event:
```

**Review by bradgrimm — COMMENTED** [review]
Looks like it is missing an and.

**Review by bradgrimm — APPROVED** [review]
Looks good.

**Pull request merged**